themes/classic/_dev/js/selectors.js

```diff
@@ -54,7 +54,7 @@ prestashop.themeSelectors = {
     searchLink: '.js-search-link',
   },
   order: {
-    returnForm: '#order-return-form, .js-order-return-form',
+    returnForm: '#order-return-form.js-order-return-form',
```

Comment:
```suggestion
    returnForm: '#order-return-form.js-order-return-form',
```

are you sure? if you want to get one element you only need `#order-return-form`, maybe you forgot to put a space between ID and class selector?

Reply:
I just let the selector with the class .js-order-return-form because maybe it was important. But indeed it would work with only with the #order-return-form